src/ast/mod.rs

```diff
@@ -473,6 +473,17 @@ pub enum Expr {
     },
     /// Unary operation e.g. `NOT foo`
     UnaryOp { op: UnaryOperator, expr: Box<Expr> },
+    /// CONVERT a value to a different data type or character encoding `CONVERT(foo USING utf8mb4)`
+    Convert {
+        /// The expression to convert
+        expr: Box<Expr>,
+        /// The target data type
+        data_type: Option<DataType>,
+        /// The target character encoding
+        charset: Option<ObjectName>,
```

Comment:
FWIW Postgres calls this "conversion name" but that appears to be an `ObjectName` so 👍 

https://www.postgresql.org/docs/8.2/functions-string.html